soapy/wfs/pyramid.py

```diff
@@ -11,187 +48,444 @@
 except NameError:
     xrange = range
 
+
 # The data type of data arrays (complex and real respectively)
 CDTYPE = numpy.complex64
 DTYPE = numpy.float32
 
+FFT_OVERSAMP = 10
+MOD_MASK = 4
 
-class Pyramid(wfs.WFS):
+def phase_screen_tip_tilt(input_array, tip_value, tilt_value):
     """
-    *Experimental* Pyramid WFS.
+    Create a phase mask of complex exponentials with modulus one and an appropriate phase at each element
+    for the given tip and tilt angles (in radians)
+    
+    Param:
+        - input_array : use to define the size of the phase screen. Typically, give the phase screen the tip-tilt aberration
```

Comment:
Can you update the doc strings to follow the google style (similar to the rest of the doc strings in the simulation), this helps with the building of the documentation.